### ISSUE TITLE
Cannot create stellargraph from networks DiGraph (AttributeError)

### ISSUE THREAD

**Issue by dproxy8** [stellargraph/stellargraph#1944]
Hi, thank you for your great work.
As the title says, I cannot create stellargraph from networks DiGraph.
I'm trying to create a stellargraph from networkx DiGraph based on [this stellargraph document.](https://stellargraph.readthedocs.io/en/latest/demos/basics/loading-networkx.html)

I could reproduce it with a simple directed graph like below:

    ```
    from stellargraph import StellarGraph as sg
    import networkx as nx
    import numpy as np
    
    
    A = np.array([[0, 1, 1, 0], [0, 0, 0, 1], [0, 0, 0, 0], [1, 0, 1, 0]])
    G = nx.from_numpy_array(A, create_using=nx.DiGraph)
    g_feature_attr = G.copy()
    features = np.array([[1, 0, 0], [0, 1, 0], [1, 0, 0], [0, 0, 1]])
    for node_id, node_data in g_feature_attr.nodes(data=True):
        node_data["feature"] = features[node_id]
    square_feature_attr = sg.from_networkx(g_feature_attr, node_type_default="paper", edge_type_default="cites", node_features="feature")
    print(square_feature_attr.info())
    
    '''StellarDiGraph: Directed multigraph
     Nodes: 4, Edges: 5
    
     Node types:
      paper: [4]
        Features: float32 vector, length 3
        Edge types: paper-cites->paper
    
     Edge types:
        paper-cites->paper: [5]
            Weights: all 1 (default)
            Features: none'''
```

However, when I tried to create with bigger graph, a cora citation network, an error occured:

**> AttributeError: module 'stellargraph' has no attribute 'from_networkx'**

I checked out all the variable's types and they are all the same, but still get the above error. 

    ```
    A = load_cora()  # cora network data adjcacency matrix ndarray (2708, 2708)
    G = nx.from_numpy_array(A, create_using=nx.DiGraph)  # cora directed graph (2708 nodes)
    
    g_feature_attr = G.copy()  # <class 'networkx.classes.digraph.DiGraph'>
    feature_path = "data/cora/cora-feats.npy"
    features = np.load(feature_path)  # feature vecotrs ndarray (2708, 1433)
        
    for node_id, node_data in g_feature_attr.nodes(data=True):
        node_data["feature"] = features[node_id]
    
    square_feature_attr = sg.from_networkx(g_feature_attr, node_type_default="paper", edge_type_default="cites",
                                               node_features="feature")
    
    print(square_feature_attr.info())
```

Basically this code is the same as the first one, differences are the `A` and `features`.
Could you provide the solution to this problem?
Thank you in advance.
